tokens/4fd0d998dc0700ca6ef89fafff05fbf523a3a25c1fc8314bf7e4d1c2.GrowthToken.json

```
@@ -0,0 +1,9 @@
+{
+  "policyId": "4fd0d998dc0700ca6ef89fafff05fbf523a3a25c1fc8314bf7e4d1c2",
+  "name": "Growth Token",
```

```suggestion
  "name": "Growth",
```

This should be the asset name. If I see correctly, the asset name for your token is just "Growth"